app/application/-components/verify-email-modal/component.ts

```diff
@@ -104,18 +104,7 @@ export default class VerifyEmailModal extends Component.extend({
 
     @computed('userEmail.isMerge')
     get translationKeys(): TranslationKeys {
-        if (!this.userEmail || !this.userEmail.isMerge) {
-            return {
-                header: 'verifyEmail.add.header',
-                body: 'verifyEmail.add.body',
-                verifyButton: 'verifyEmail.add.verifyButton',
-                denyButton: 'verifyEmail.add.denyButton',
-                verifySuccess: 'verifyEmail.add.verifySuccess',
-                denySuccess: 'verifyEmail.add.denySuccess',
-                verifyError: 'verifyEmail.add.verifyError',
-                denyError: 'verifyEmail.add.denyError',
-            };
-        } else {
+        if (this.userEmail && this.userEmail.isMerge) {
```

Comment:
Strictly speaking, this is outside the scope of this PR, but it is much more clear, so might as well let it through.